Platform/ElkhartlakeBoardPkg/Library/Stage1BBoardInitLib/Stage1BBoardInitLib.c

```diff
@@ -760,8 +765,10 @@ PlatformFeaturesInit (
     // Update platform specific feature from configuration data.
     PlatformFeatures           = &((PLATFORM_DATA *)GetPlatformDataPtr ())->PlatformFeatures;
     if (PlatformFeatures != NULL) {
-      if (FeaturePcdGet (PcdPreOsCheckerEnabled) && PchIsSciSupported ()) {
+      if (FeaturePcdGet (PcdPreOsCheckerEnabled) && PchIsSciSupported () && !FixedPcdGetBool (PcdIsReleaseBuild)) {
         PlatformFeatures->DebugConsent = 7;
+      } else if (FixedPcdGetBool (PcdIsReleaseBuild)){
+        PlatformFeatures->DebugConsent = 0;

```

Comment:
What is default PlatformFeatures->DebugConsent value if FeaturesCfgData is NULL or PlatformFeatures is NULL? Can you initialize default PlatformFeatures->DebugConsent at the beginning of code?

Reply:
Ok noted